README.md

```diff
@@ -55,61 +20,55 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 David's Portfolio <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
```

Comment:
Please, remove every instruction from your code, instructions are there to guide you on what to do.